Cargo.toml

```diff
@@ -36,6 +36,9 @@ base64 = "0.22.1"
 xml-rs = "0.8.4"
 zstd = { version = "0.13.1", optional = true, default-features = false }
 flate2 = "1.0.28"
+serde = "1.0.215"
+serde_json = "1.0.133"
+regex = "1.11.1"
```

Comment:
These dependencies are pretty heavy, maybe we should feature-gate world file parsing (under a `world` feature perhaps)? It's not something that everyone uses, and it's in a completely different format to TMX files

Reply:
I've added a crate feature "world" the will be in when I update the PR after addressing the other issue/topics. Thanks!